### PR TITLE
This branch is to add the Feat: Create book and Remove book using redux

### PR DESCRIPTION
**Pull request by steveWDamesJr**
## Add Functionality - Create book and Remove Book

-Configure Redux Store
-Write actions and Reducers for books
-Set Data inputs in the Local React state
-Dispatch corresponding actions in deleteBook
-Implement AddBook Logic
-Implement Remove Book Logic
-Check application with Redux-Logger

-Debug Linters

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import booksReducer from './books/books';
 
 const reducer = combineReducers({
-  todo,
+  booksReducer,
 });
 
-const configureStore = (initialState) => createStoreWithMiddleware(reducer, initialState);
-export default configureStore;
+const composeEnhancements = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__;
```
Comment (dasileker) [inline src/redux/configureStore.js]: - [x] this line number 9 i don't see how `window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__;` is fit??

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @steveWDamesJr,** 

### **Good job so far!**
You still need to work on some issues to go to the next project, but you are almost there!

## **Highlights**
- The PR is good👌.
- Linters are passing 👌.
- The Readme file is good👌

## Required Changes ♻️
_Check the comments under the review._

- Nice job on adding redux however the page is not displayed correctly.


Capture d’écran 2022-03-17 100447


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (steveWDamesJr) [reply src/redux/configureStore.js]: Thank you for your kind reply.

https://user-images.githubusercontent.com/81635839/158825620-b66cb8e0-43f5-4ee2-86de-985298f7b640.mp4


Please see resolution to errors.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @steveWDamesJr 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**